build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+driver_images_tag_from_branch=develop
+DOCKER_HUB_USERNAME=csiblock1
```

Comment:
shouldn't this also be a secret?

Reply:
since it's in the PR, I think even if we make it a secret, it's already public.

Reply:
I don't think that's a reason to keep it (and usernames can always be changed)

Reply:
I agree

Reply:
how it's different from other parameters that you didn't ask to put as a secret?
IMO secrets are meant only for passwords and staff like this

Reply:
I think in general, credentials are considered secrets, i.e. both username and password.
at least that's what I saw in [k8s](https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret)

Reply:
Username is a part of the credentials as Ori mentioned.